src/cmake/install_nix.cmake

```
@@ -13,14 +13,14 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
     )
 
     # install icons
-    foreach(res IN ITEMS 16 32 48 64 128)
+    foreach(res IN ITEMS 16 32 48 64 128 256 512)
         file(
             COPY ${CMAKE_CURRENT_SOURCE_DIR}/install/nix/blue-nebula_x${res}.png
-            DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/icons/hicolor/${res}/apps/
+            DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/icons/hicolor/${res}x${res}/apps/
         )
         file(RENAME
-            ${CMAKE_CURRENT_BINARY_DIR}/icons/hicolor/${res}/apps/blue-nebula_x${res}.png
-            ${CMAKE_CURRENT_BINARY_DIR}/icons/hicolor/${res}/apps/blue-nebula.png
+            ${CMAKE_CURRENT_BINARY_DIR}/icons/hicolor/${res}x${res}/apps/blue-nebula_x${res}.png
+            ${CMAKE_CURRENT_BINARY_DIR}/icons/hicolor/${res}x${res}/apps/blue-nebula.png
             )
```

While we're at it, mind to fix the code style?

```suggestion
        )
```